src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -12,9 +12,7 @@
             [utils.re-frame :as rf]
             [i18n.i18n :as i18n]
             [quo2.core :as quo]
-
-            ;; TODO move to status-im2
-            [status-im.ui.screens.communities.community :as community]))
+            [status-im.ui.screens.communities.community-options-bottom-sheet :as home-actions]))
```

Comment:
please add comments back, because its still should be implemented in 2 namespace

Reply:
oh yeah